### ISSUE TITLE
Ingame videos stuttering/freezing entirely

### ISSUE THREAD

**Issue by Brawdogger** [weathondev/SingstarCreatorTutorial#9]
Whilst testing some of the songs I've managed to get working, I have noticed that whilst ingame, some of their videos stutter quite heavily and freeze entirely. On first glance I reckon it might have something to do with the overall file size of some of these videos and may have to test to see if lowering the bitrate/lowering the resolution/Converting to a different file type will improve them

**Comment by weathondev** [comment]
Yeah it might be bitrate, resolution or whatever, so far i haven't found out what the reason could be, but as video was never the priority i haven't invested much time.
